src/controllers/user-role-settings.controller.js

```diff
@@ -0,0 +1,48 @@
+import ResponseService from '../services/response.service';
+import UserService from '../services/user.service';
+import JwtService from '../services/jwt.service';
+
+/**
+ *
+ *
+ * @class UserRoleSettingsController
+ */
+class UserRoleSettingsController {
+  /**
+  * super_admin can assign a role to user by email
+  * @static
+  * @description PATCH /api/admin/reset-user-role
+  * @param {object} req request object
+  * @param {object} res response object
+  * @memberof UserRoleSettingsController
+  * @returns {object} ResponseService
+  */
+  static async changeUserRole(req, res) {
+    const userToken = JwtService.verifyToken(req.headers.authorization);
+    const userData = await UserService.findUserByProperty({ email: userToken.email });
+    if (userData.role === 'super_admin') {
+      const { userEmail, newRole } = req.body;
+      const userExist = await UserService.findUserByProperty({ email: userEmail.trim() });
+      if (userExist && userExist.role !== 'super_admin') {
+        if (userExist.role === newRole.trim()) {
+          ResponseService.setError(400, `User Role Not Updated. ${userExist.email} is already a ${userExist.role}`);
+          ResponseService.send(res);
+        } else {
+          await UserService.updateUser({ email: userEmail.trim() }, { role: newRole.trim() });
+          const data = await UserService.findUserByProperty({ email: userEmail.trim() });
+          const { password, token, createdAt, updatedAt, isVerified, ...datas } = data.dataValues;
+          ResponseService.setSuccess(200, 'User role successfully updated', datas);
```

Comment:
Since you updated only the user's role, you should only send that as a response.